VOUnits.tex

```diff
@@ -412,6 +416,81 @@ \subsection{What this document will not do}
 with scaling prefixes (cf \prettyref{sec:scalingprefixes}).
 \end{itemize}
 
+\subsection{VOUnits syntax summary\label{sec:cribsheet}}
+
+This section provides an informal cribsheet for the most important
+rules defined by VOUnits syntax, which is recommended for use
+within the VO.
+More detailed rules and explanations, as well as special cases
+and less obvious consequences,
+are provided in the body of \prettyref{sec:proposal},
+and the syntax is formally defined in \prettyref{appx:vougrammar}.
+\begin{itemize}
+\item Unit strings are printable ASCII characters (\prettyref{sec:encoding})
+\item Unit strings are case-sensitive (\prettyref{sec:encoding})
+\item No whitespace is permitted (\prettyref{appx:vougrammar})
+\item Unit names are, mostly, represented as you would expect.
+      Most unit base symbols required by astronomy data are recognised,
+      including:
```

Comment:
Perhaps here, or in the sublist, or even in the first para of the subsection, include a reference to \prettyref{tabx:knownunits}, since that seems the sort of thing it'd be useful to simply look up from time to time.